packages/admin/components/forms/FormGrid/FormGrid.styles.ts

```diff
@@ -21,4 +22,19 @@ export const Grid = styled.div`
   fieldset & {
     padding-block-end: 0;
   }
+
+  > fieldset + div {
+    border-top: 1px solid var(--neutral40);
+    padding-block-start: ${pxToRem(40)};
+    margin-block-start: calc(${pxToRem(40)} - var(--form-grid-row-gap));
+
+    ${respond(
+      `
+        padding-block-start: ${pxToRem(48)};
+        margin-block-start: calc(${pxToRem(48)} - var(--form-grid-row-gap));
+
+      `,
+      50
+    )}
```

Comment:
@jen-castiron is there a better way to handle to specific spacing here? See: https://www.figma.com/file/EeaBT8NWvguKGhMQ7pgpry/NGLP-Admin-UI-Design?node-id=2007%3A11733